macros/general/troubleshooting-ticket.mdx

```diff
@@ -0,0 +1,29 @@
+---
+macro: troubleshooting-ticket
+---
+
+If you did not manage to identify the error and solve it by yourself, [open a support ticket](/account/how-to/open-a-support-tickets/#writing-an-effective-subject-and-description), and provide as many details as possible, along with the necessary information below:
+    - Resource ID
+    - Resource name
+    - Request type (PUT, GET, etc.)
```

Comment:
Since request type and HTTP status code will not be relevant to all issues, I'd move them to the bottom of the list with an (if applicable) after each